### PR TITLE
Onboard openshift-cnv/cnv-ci

### PR DESCRIPTION
**Pull request by lmilbaum**

### CODE REVIEW

File: core-services/prow/02_config/_plugins.yaml [hunk 1]
```diff
@@ -2074,6 +2074,8 @@ plugins:
   - welcome
   - wip
   - yuks
+  openshift-cnv/cnv-ci:
```
Comment (droslean) [inline core-services/prow/02_config/_plugins.yaml]: it is better to have a config for the whole organization

Comment (lioramilbaum) [reply core-services/prow/02_config/_plugins.yaml]: I have updated the file. The webhooks and the robots are configured at the moment on repo level. cnv-ci is our only repo at this moment.

**Pull request merged**